src/passim-integration-test.py

```
@@ -0,0 +1,645 @@
+    def _start_daemon(cls, builddir):
+        """Start the passimd binary."""
+        daemon_path = os.path.join(builddir, "src", "passimd")
+        env = os.environ.copy()
+        env["DBUS_SYSTEM_BUS_ADDRESS"] = os.environ.get(
+            "DBUS_SYSTEM_BUS_ADDRESS", ""
+        )
+        env["LOGS_DIRECTORY"] = cls.log_path
+        env["G_MESSAGES_DEBUG"] = "all"
+        env["G_DEBUG"] = ""
+
+        cls.daemon_proc = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)